README.md

```diff
@@ -15,6 +15,57 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
   * Gluesync  - A metastore event listener to replay Hive metadata events in a Glue catalog.
   * Metastore authorization - A metastore pre-event listener to handle authorization using Ranger.
 
+## Variables
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| apiary_customer_accounts | aws account ids for clients of this metastore | list | - | yes |
+| apiary_database_name | Database name to create in RDS for Apiary | string | `apiary` | no |
+| apiary_domain_name | Apiary domain name for route 53 | string | `` | no |
+| apiary_log_bucket | bucket for apiary logs | string | - | yes |
+| apiary_log_prefix | prefix for apiary logs | string | `` | no |
+| apiary_managed_schemas | schema names from which s3 bucket names will be derived,corresponding s3 bucket will be named as apiary_instance-aws_account-aws_region-schema_name | list | `<list>` | no |
+| apiary_producer_iamroles | aws iam roles allowed write access to managed apiary s3 buckets | map | `<map>` | no |
+| apiary_rds_additional_sg | Comma-seperated string for additional security groups to attach to RDS | list | `<list>` | no |
+| apiary_s3_alarm_threshold | will trigger cloudwatch alarm if s3 is greater than this, default 1TB | string | `10000000000000` | no |
+| apiary_tags | Common tags that get put on all resources | map | - | yes |
+| aws_region | aws region | string | - | yes |
+| db_backup_retention | The days to retain backups for, for the rds metastore. | string | - | yes |
```

Comment:
The days to retain backups for, for the rds metastore. -> Number of days to retain metastore RDS backups for